AirCasting/Utils/Bluetooth/NewBluetoothManager.swift

```diff
@@ -374,6 +374,7 @@ final class NewBluetoothManager: NSObject, BluetoothCommunicator, CBCentralManag
     }
     
     func peripheral(_ peripheral: CBPeripheral, didUpdateValueFor characteristic: CBCharacteristic, error: Error?) {
+        Log.verbose("New value for characteristic: \(characteristic.value)")
```

Comment:
Do we need it at the end? If so maybe `Log.info`?

Reply:
No, we don't need this. Thanks :)